packages/messaging-api-slack/src/SlackOAuthClient.ts

```diff
@@ -120,56 +114,66 @@ type ScheduleMessageOptions = CommonOptions &
     postAt?: string;
   };
 
-type PostMessageOptions = PostMessageOptionalOptions &
+export type PostMessageOptions = PostMessageOptionalOptions &
   Message & {
     channel: string;
   };
 
-type PostEphemeralOptions = PostEphemeralOptionalOptions &
+export type PostEphemeralOptions = PostEphemeralOptionalOptions &
   Message & {
     channel: string;
     user: string;
   };
 
-type GetScheduledMessagesOptions = CommonOptions & {
+export type GetScheduledMessagesOptions = CommonOptions & {
   channel?: string;
   cursor?: string;
   latest?: string;
   limit?: number;
   oldest?: string;
 };
 
-type UnfurlOptions = CommonOptions & {
+export type UnfurlOptions = CommonOptions & {
   ts: string;
   unfurls: {};
   userAuthMessage?: string;
   userAuthRequired?: boolean;
   userAuthUrl?: string;
 };
 
-type OpenViewOptions = {
+export type OpenViewOptions = {
   triggerId: string;
   view: View;
 };
 
-type PublishViewOptions = {
+export type PublishViewOptions = {
   userId: string;
   view: View;
   hash?: string;
 };
 
-type UpdateViewOptions = {
+export type UpdateViewOptions = {
   view: View;
   externalId?: string;
   hash?: string;
   viewId?: string;
 };
 
-type PushViewOptions = {
+export type PushViewOptions = {
   triggerId: string;
   view: View;
 };
 
+export type GetPermalinkOptions = CommonOptions & {
+  channel: string;
+  messageTs: string;
+};
+
+export type MeMessageOptions = CommonOptions & {
```

Comment:
IMO, moving those types to SlackTypes is the better way to export them. See: https://github.com/Yoctol/messaging-apis/blob/12133719c7ae1266347c3f6111873fc80157bbdc/packages/messaging-api-slack/src/index.ts#L3

Reply:
@darkbtf What do you think?

Reply:
Sounds reasonable.  I'll move those to SlackTypes